R/istr.R

```diff
@@ -38,13 +39,17 @@ istr <- function(companies, scenarios, mapper) {
 
 #' @rdname istr
 #' @export
-istr_at_product_level <- function(companies, scenarios, mapper) {
+istr_at_product_level <- function(companies,
+                                  scenarios,
+                                  mapper,
+                                  low_threshold = 30,
```

Comment:
I am not familiar with istr so I do not know - but just wanted to flag that here the thresholds are between 1-100 not like pstr. if this is correct my comment can be ignored ! :)